src/providers/state.js

```diff
@@ -0,0 +1,11 @@
+angular.module('bicker_router').provider('State', function() {
+
```

Comment:
Just return the object instead of giving it a named variable, assigning to it, then returning it via name - I'll stop commenting on these, please find and fix them all